test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2722,6 +2738,22 @@ Describe "Invoke-RestMethod tests" -Tags "Feature", "RequireAdminOnWindows" {
         Get-Item $outFile | Select-Object -ExpandProperty Length | Should -Be $content.Content.Length
     }
 
+    It "Invoke-RestMethod -OutFile folder Downloads the file and names it as the Content-Disposition header if present." {
+        $ContentDisposition = [System.Net.Http.Headers.ContentDispositionHeaderValue]::new("attachment")
+        $ContentDisposition.FileName = 'DownloadedFile.txt'
+
+        $uri = Get-WebListenerUrl -Test 'ResponseHeaders' -Query @{ "Content-Disposition" = $ContentDisposition }
```

Comment:
should add a test variation that includes full path and quotes

Reply:
@SteveL-MSFT done